errdefs/http_helpers.go

```diff
@@ -100,10 +100,10 @@ func FromStatusCode(err error, statusCode int) error {
 			err = System(err)
 		}
 	default:
-		logrus.WithFields(logrus.Fields{
+		logrus.WithError(err).WithFields(logrus.Fields{
 			"module":      "api",
-			"status_code": fmt.Sprintf("%d", statusCode),
-		}).Debugf("FIXME: Got an status-code for which error does not match any expected type!!!: %d", statusCode)
+			"status_code": statusCode,
+		}).Debugf("FIXME: Got an status-code for which error does not match any expected type!!!")
```

Comment:
"post-LGTM nit": this could now be `Debug` instead of `Debugf` (not a blocker for me, but feel free to update if you think it's cleaner)

Reply:
changed, haha I forgot that I removed all the parameters for `Debugf`, thanks for pointing out